src/messages/AdaptiveCards/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { useCallback } from "react";
+import classnames from "classnames";
+
+import AdaptiveCard from "./components/Adaptivecard";
+import { useMessageContext } from "../hooks";
+import styles from "./styles.module.css";
+
+import type { IMessage } from "@cognigy/socket-client";
+import ChatBubble from "src/common/ChatBubble";
+import { Action, HostConfig } from "adaptivecards";
+
+const adaptiveCardsHostConfig: HostConfig = {
+	fontFamily: "inherit",
+	fontSizes: {
+		small: 10,
+		default: 14,
+		medium: 16,
+		large: 18,
+		extraLarge: 34,
+	},
+	fontWeights: {
+		lighter: 300,
+		default: 400,
+		bolder: 600,
+	},
+	lineHeights: {
+		small: 12,
+		default: 18.2,
+		medium: 22.4,
+		large: 23.4,
+		extraLarge: 40.8,
+	},
+
+	containerStyles: {
+		default: {
+			backgroundColor: "#fff",
+			foregroundColors: {
+				//@ts-ignore
+				default: {
+					default: "var(--black-10)",
+					subtle: "var(--black-40)",
+				},
+			},
+		},
+	},
+};
+
+export const AdaptiveCards = () => {
+	const { message, action: onSendMessage, config } = useMessageContext();
+
+	const getCardPayload = (message: IMessage) => {
+		//@ts-ignore
```

Comment:
we should refine the TS declarations at some point.

Reply:
This is not an intentional error. Like the one which is explicitly false, which you would use in e.g. test case :)

We should do a next round of interfaces improvements at some point.